src/powerbidedicated/azext_powerbidedicated/azext_metadata.json

```diff
@@ -0,0 +1,5 @@
+{
+    "azext.isPreview": true,
+    "azext.minCliCoreVersion": "2.0.67",
```

Comment:
have you tested in 2.0.67? May it be to low?

Reply:
There are some breaking changes for 2.0.67 and previous versions including some new features only available after 2.0.67.